webcompat/static/js/lib/bugform.js

```diff
@@ -550,7 +553,8 @@ function BugForm() {
             "The image is too big! Please choose something smaller than 4MB.";
           wcEvents.trigger("flash:error", { message: msg, timeout: 5000 });
         }
-        this.loadingIndicator.hide();
+
+        this.loadingIndicator.hide;
```

Comment:
Won't this make the screenshot upload thingy disappear now?

`<div class="form-upload js-image-upload js-Form-group loader js-loader js-image-loader">`